reccmp/tools/asmcmp.py

```diff
@@ -322,15 +332,20 @@ def main():
 
     implemented_funcs = function_count
 
+    # Substitute an alternate value for the total number of functions in the file
     if args.total:
         function_count = int(args.total)
 
     if function_count > 0:
         effective_accuracy = total_effective_accuracy / function_count * 100
         actual_accuracy = total_accuracy / function_count * 100
+        alignment_accuracy = functions_aligned_count / function_count * 100
```

Comment:
Maybe `alignment_percentage`? I think `accuracy` is fine for the other variables since the code can be matched to a fractional degree, but alignment is binary

Reply:
Yes 👍